tests/tracking/fluent/test_fluent_autolog_logger.py

```diff
@@ -0,0 +1,11 @@
+import mlflow
+from unittest import mock
+
+
+def test_autolog_success_message_suppressed_when_disabled():
+    with mock.patch("mlflow.tracking.fluent._logger.info") as autolog_logger_mock:
+        mlflow.autolog(disable=True)
+        # pylint: disable=unused-variable
+        import tensorflow
+
+        autolog_logger_mock.assert_not_called()
```

Comment:
Had a lot of trouble adding this to `test_fluent_autolog.py` because `import tensorflow` was already at the top of the file, so the import hooks wouldn't trigger on re-import. Would like some pointers on working around that if any

Reply:
test_fluent_autolog.py has utilities for this. We should move this test case to that file and use the utilities present there, instead of creating a new test suite

Reply:
Helped offline + done